run

```diff
@@ -47,32 +40,54 @@ fi
 
 # Parse arguments
 UPDATE_ENV=false
+LOCAL=false
 ARGS=()
 
 for arg in "$@"; do
     case $arg in
-        --update-env)
+        update-env)
         UPDATE_ENV=true
         ;;
-        *)
-        ARGS+=("$arg")
+        local)
+        LOCAL=true
         ;;
     esac
+    ARGS+=("$arg")
 done
 
-# Inject secrets to .env if the flag is set
 if [ "$UPDATE_ENV" = true ]; then
-    # Check if 1Password CLI is installed
     if ! which op > /dev/null; then
-        echo "The MDCT team uses 1Password to store and retrieve development credentials. If you do not have a 1Password account please request one."
```

Comment:
We should still have this note to request a 1password account if you don't have one, right?

Reply:
My 2c is we should either be specific or remove it. Like what is an email address they should send a message to to request a 1password account? As is we're being too vague to be helpful to a real person. What do you think?

Reply:
I don't think its a big deal either way.  I think the guidance would have to be if you don't have a 1password account, ask your on-boarding lead, which is what I hope they will do if we leave them hanging with no guidance :)

Reply:
Yeah my thoughts exactly. I'll leave it out for now and if people get hung up we can hassle some folks for a more specific message.